client/src/updates_view.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Update from './update.jsx';
+import styled from 'styled-components';
+
+const Background = styled.div`
+	font-family: "Motiva Sans", Sans-Serif;
+	background-color: #1b2838;
+	width: 616px;
+	height: 262.4px;
+	margin-top: 30px;
+	color: white;
+	display: inline-block;
+`
+
+const Header = styled.div`
+	background-image: url(https://steamstore-a.akamaihd.net/public/images/v6/maincol_gradient_rule.png);
+	background-repeat: no-repeat;
+	background-position: bottom left;
+	padding-top: 2px;
+	margin-bottom: 10px;
+	height: 25.6px;
+	font-size: 14px;
+	color: white;
+	letter-spacing: 2px;
+	line-height: 26px;
+`
+
+const Button = styled.button`
+	border: 1px solid rgba(255, 255, 255, 0.4);
+	border-radius: 2px;
+	font-size: 11px;
+	color: white;
+	background-color: transparent;
+	float: right;
+	padding: 2px 15px;
+	margin-right: 2px;
+	&:hover {
+		border: 1px solid white;
+	}
+`
+
+const Update_divider = styled.div`
+	width: 5%;
+	font-size: 14px;
+	height: 190px;
+	display: inline-block;
+`
+
+const View = (props) => (
+
+	<div className="view">
+		<Background>
+			<Header>
+				RECENT UPDATES
+				<Button>
+					View all ({props.updates.length})
+				</Button>
+			</Header>
+			<Update story={props.updates[0]}/>
```

Comment:
It's recommended to not hardcode array indexes like this, since it's possible in the future you'll have an updates array with N number of items.

Should be doing something like 
```javascript
</header>
{
  props.updates.map(update => <Update story={update} />)
}
<Update_divider />

```

Or if updates is a tuple, e.g.  [0] is a name and [1] is a value, it's better to either use an object literal, or to pass each individually as a prop. Example
```javascript
// When invoking View component
<View name={this.state.updates[0]} value={this.state.updates[1]} />
...
// In View component
const View = ({name, value) => (
<Update story={name}/>
<Update story={value}/>
```

Reply:
Well I did that because it only displays the first two.. or at least the two most recent which I am currently just treating as the first two in the array. I probably should write a conditional for in case there is less than 2 updates. Could also write logic to fetch 2 updates from the db with the most recent post date but given dummy data that didn't seem necessary

Reply:
Ahh okay I see what you're doing now - they'd only have at most 2 articles displayed. 

I'd still consider passing each article as a prop just for whoever will be working on the BE to easily understand. 
```javascript
<View firstArticle={this.state.updates[0]} secondArticle={this.state.updates[1]} />
```

Then
```javascript
const View = ({firstArticle, secondArticle}) => (
  <Update story={firstArticle}/>
  <Update story={secondArticle}/>
);
```

Not blocking or anything though - can consider anytime. Looks good 👍